task1_string_manipulation/helper.py

```diff
@@ -0,0 +1,52 @@
+
+def removeMultyOccurence(str):
+    temp = ""
+
+    for s in str:
+        if s.lower() not in temp.lower():
+            temp = temp + s
+
+    return temp
+
+def repeats(str, ch):
+    count = 0
+    for s in str:
+        if s.lower() == ch.lower():
+            count = count + 1
+
+    if count>1:
+        return True
+    else:
+        return False
+
+def removeFirst(str, ch):
```

Comment:
Can not differentiate between functionality of functions `removeFirst` and `removeFirstOccurence` because the names are alike. Please use more descriptive names.